src/IUsdPlusRedeemer.sol

```diff
@@ -91,6 +92,11 @@ interface IUsdPlusRedeemer {
         external
         returns (uint256 ticket);
 
+    /// @notice Allows rescue of USD+ tokens that are stuck in the contract
+    /// @param to The address to send the rescued USD+ tokens to
+    /// @param amount The amount of USD+ to rescue
+    function rescueFunds(address to, uint256 amount) external;
```

Comment:
Add a @dev warning that improper use can lead to unfillable requests due to insufficient USD+